assets/dev/scss/frontend/widgets/link-in-bio.scss

```diff
@@ -79,6 +154,7 @@
 			border-style: var(--e-link-in-bio-identity-image-profile-border-style);
 			border-width: var(--e-link-in-bio-identity-image-profile-border-width);
 			overflow: hidden;
+			margin-block-end: pxToRem(17px);
```

Comment:
Just a question, why do we need this `pxToRem`? I haven't really seen it in any of the widgets before 🤔

Reply:
@louiswol94 I added it purely because it was in use in a couple of widgets I was referencing :-) I can remove it if we prefer px values to rem.

Reply:
@willhowat If there was no specific reason for it, please remove it. Less things that can break. Thanks :)

Reply:
Done